02-express-tutorial/app.js

```diff
@@ -3,22 +3,40 @@
 const express = require('express');
 const { products } = require('./data');
 const app = express();
+const people = require('./data.js')
+const peopleRouter = require('./routes/people');
 
-app.use(express.static("./public"));
 
-app.get('/api/v1/test', (req, res) => {
+const logger = (req, res, next) => {
+    const currentTime = new Date().toISOString();
+    console.log(`[${currentTime}] ${req.method} ${req.url}`);
+    next();
+};
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use(logger);
+
+app.use(express.static("./methods-public"));
+
+app.get('/', logger, (req, res) => {
+    res.send('Welcome to the home page');
+});
+
+app.get('/api/v1/test', logger, (req, res) => {
     res.json({ message: 'It worked!' });
 });
 
-app.get('/api/v1/test', (req, res) => {
+app.get('/api/v1/test', logger, (req, res) => {
     res.json(products);
 });
 
-app.get('/api/v1/test', (req, res) => {
+app.get('/api/v1/test', logger, (req, res) => {
```

Comment:
This `app.get('/api/v1/test', ...);` block should be removed, since line 27 already handles this path.